rsts/deployment/plugins/webapi/databricks.rst

```diff
@@ -381,3 +381,54 @@ Wait for the upgrade to complete. You can check the status of the deployment pod
   Make sure you enable `custom containers 
   <https://docs.databricks.com/administration-guide/clusters/container-services.html>`__
   on your Databricks cluster before you trigger the workflow.
+
+Databricks Agent Service configuration
+--------------------------------------
+
+You can use Spark via the Databricks Agent Service.
+
+To do so, follow these steps:
+
+1. Start the agent service using PyFlyte.
```

Comment:
Most people use the agent in Kubernetes, so we don't need to mention `python serve` here.

Reply:
Sure, no problem!